docker-compose.yml

```diff
@@ -29,12 +64,16 @@ services:
     container_name: shoe_project_pgadmin
     image: dpage/pgadmin4
     environment:
-      PGADMIN_DEFAULT_EMAIL: user@blueprint.com
-      PGADMIN_DEFAULT_PASSWORD: admin
+      PGADMIN_DEFAULT_EMAIL: 'user@blueprint.com'
```

Comment:
`uwblueprint.org` is more correct here for us, but I guess it doesn't really matter.

Reply:
yeah i did not realize it was `org` lol will fix